dev-scripts/dev.js

```diff
@@ -37,7 +37,7 @@ function startServer () {
           resolve()
         } else {
           if (!firstRun) {
-            console.log(stats.compilation.errors[0])
+            return
```

Comment:
That console.log is actually there by design :smile: Please undo you change :smile: